vendors/polysense/drivers/WxS8800-362021/driver.yaml

```diff
@@ -0,0 +1,17 @@
+# Formal driver name
+name: Driver Polysense
+# Simple description of the driver functionalities
+description: The WxS8800-362021 is a LoRaWAN Sound/Noise Smart Sensor & RTU (Remote Terminal Unit) 2-in-1 Terminal.
```

Comment:
A driver description not model. Something like: A driver to decode payload of Polysense WxS8800-362021 device.